frontend/src/views/HomePage.vue

```diff
@@ -28,5 +30,7 @@ export default {
   <div class="home">
     <PageGreeting msg="This is the Home Page" />
     <button @click="handleLogOut">Logout</button>
+    <hr />
+    <constraint-form />
```

Comment:
Please be consistent with the naming of custom component tags. A tag a couple of tags above is written in camel case while this one is in kebab-case.

Reply:
name above was added in vue setup commit, I didn't want to touch it now because it is not related but it will be fixed in separate PR